console_conf/core.py

```diff
@@ -23,9 +23,9 @@
 class ConsoleConf(Application):
 
     project = "console_conf"
-    controllers = {
-        "Welcome": None,
-        "Network": None,
-        "Identity": None,
-        "Login": None,
-    }
+    controllers = [
```

Comment:
I don't know the code, but I notice that this is changing a dict with None values to a list.  If order doesn't matter, perhaps a set would work better?